packages/storybook/src/react/DotIndicator.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Meta, StoryObj } from '@storybook/react';
+import { Button } from '@gemeente-denhaag/button';
+import { DotIndicator } from '@gemeente-denhaag/dotindicator';
+import readme from '../../../../components/DotIndicator/README.md';
+
+const exampleArgs = {
+  overlap: 'rectangle',
+};
+
+const meta = {
+  id: 'react-data-display-dot-indicator',
+  title: 'React/Data Display/Dot indicator',
+  component: DotIndicator,
+  args: exampleArgs,
+  tags: ['autodocs'],
+  parameters: {
+    chromatic: { viewports: [1768, 1280, 768, 360] },
```

Comment:
Eerst deden we voor componenten geloof ik enkel tests op een of twee resoluties. @Yolijn Is het een probleem qua kosten als we alles op vier resoluties gaan testen? Of is Chromatic gratis omdat het een open source project is?

Indien het geen probleem is, dan kunnen we die chromatic parameter waarschijnlijk beter als default toevoegen aan de preview.tsx in storybook.

Reply:
Hmm ja dat gaat wel significante kosten opleveren... we zouden ze daarom volgens mij voor de viewports van alle template pagina's doen?

Reply:
We krijgen de eerste 35000 screenshots gratis (1 screenshot = 1 story van 1 viewport) maar dat is voor de hele nl-design-system organisatie samen.

Reply:
oh en we doen de benodigde viewports voor componenten die een responsive variant hebben zoals bijvoorbeeld de card-group

Reply:
@Borai Dan zou ik inderdaad de default viewport op 1280 zetten (geloof dat dat eerst ook de default was), en dan voor componenten die op andere viewports significant anders zijn de viewport overriden in de story.